src/webots/control/WbController.cpp

```diff
@@ -314,6 +321,9 @@ void WbController::start() {
   }
   file.write("");
   file.close();
+
+  // FIXME: from Qt 6.4 onwards, QDir::mkdir can be used to set the permissions
+  QProcess::execute("sh", QStringList() << "-c" << QString("chmod -R 777 %1").arg(mIpcPath));
```

Comment:
This code seems wrong: useless and not working on Windows where it is supposed to be executed.